rust/xaynet-server/src/state_machine/phases/idle.rs

```diff
@@ -21,22 +19,14 @@ use sodiumoxide::crypto::hash::sha256;
 #[derive(Debug)]
 pub struct Idle;
 
-#[async_trait]
-impl Handler for PhaseState<Idle> {
```

Comment:
I'm surprised we don't need that. Don't we have a trait bound somewhere that looks like `PhaseState<T>: Handler`? Or maybe we use to have it.

Reply:
yep you are right 
there is a trait bound in https://github.com/xaynetwork/xaynet/blob/30cb9cb180e3f1c1b2c5b8bd04c9dc2ed36df5cd/rust/xaynet-server/src/state_machine/tests/builder.rs#L38-L42
I will remove it 🙂